tdrs-backend/tdpservice/scheduling/tasks.py

```diff
@@ -0,0 +1,32 @@
+"""Tasks that are triggered by the scheduler."""
+from __future__ import absolute_import
+from tdpservice.users.models import User
+
+from django.utils import timezone
+from celery import shared_task
+from datetime import datetime, timedelta
+import logging
+from tdpservice.email.email_helper import send_deactivation_warning_email
+
+
+logger = logging.getLogger(__name__)
+
+@shared_task
+def check_for_accounts_needing_deactivation_warning():
+    """Check for accounts that need deactivation warning emails."""
+    deactivate_in_10_days = User.objects.filter(
```

Comment:
Another thought I had for these queries is that they could check for accounts that aren't already in a deactivated status, might not be necessary but 🤷